packages/shared/src/models/IRouterState.ts

```diff
@@ -0,0 +1,21 @@
+/**********************************************************************
```

Comment:
AFAIK it should not be a shared information as it's only on the frontend side ?

Reply:
Yes